website/docs/compute/local-deployment/minikube.md

```diff
@@ -0,0 +1,225 @@
+# Local Deployment with Minikube
+
+It’s possible to deploy a Compute environment to a local environment using a local Kubernetes cluster.
+
+## Requirements
+
+- [Docker](https://www.docker.com/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [ngrok](https://ngrok.com/download)
+- [subo](https://github.com/suborbital/subo#installing)
+
+## Steps
+
+### 1. Create a folder for the environment
+
+This is a temporary place where we’ll create and configure our Compute environment:
```

Comment:
Does it matter *where* this directory is created?

Reply:
Nope!